### PR TITLE
Bump version number to v0.14-dev

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+### New features
+
+### Improvements
+
+### Bug fixes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order
```
Comment (josh146) [inline CHANGELOG.md:11]:
```suggestion
This release contains contributions from (in alphabetical order):
```

**Review by thisac — APPROVED** [review]
:muscle:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Release 0.14.0-dev
+
+### New features
+
+### Improvements
+
+### Bug fixes
+
+### Contributors
```
Comment (thisac) [inline CHANGELOG.md:9]: Oh, should there be a documentation here as well?
```suggestion
### Bug fixes

### Documentation

### Contributors
```

Comment (josh146) [reply CHANGELOG.md:9]: Oops, good catch!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# Release 0.14.0-dev
+
+### New features
```
Comment (thisac) [inline CHANGELOG.md:3]: Prior, it said, `### New features since last release`. I think I prefer this though. :thinking:

Comment (josh146) [reply CHANGELOG.md:3]: Let's leave it as is!

**Pull request merged**